CI.md

```diff
@@ -75,8 +75,27 @@ run categories. Here is a summary of the run categories with regards of the jobs
 Those jobs often have matrix run strategy which runs several different variations of the jobs
 (with different platform type / Python version to run for example)
 
+### Google Cloud Platform Credentials
+
+Some of the jobs require variables stored as a [GitHub Secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets)
+to perform operations on Google Cloud Platform. Currently these jobs are limited to Apache repository only.
```

Comment:
A user can set these secrets themselves though to test on their own fork?

Reply:
Yes, if variables are set workflow can be fully tested on the fork. It will be much more easier thanks to manual trigger by `workflow_dispatch` event which provide button to do this. Sentence `Currently these jobs are limited to Apache repository only` is not valid since GCP variables check was introduced. I missed that. I will create separate PR for this fix. Thanks for pointing this out.

Reply:
https://github.com/apache/beam/pull/12598